### DISCUSSION TITLE
Auto-crafting Upgrade

### DISCUSSION THREAD

**Discussion by Mad-Matti** [P3pp3rF1y/SophisticatedBackpacks#304] [Ideas]
Similarly to the Auto-smelting Upgrade, the Auto-crafting upgrade would allow you to set items for the bag to automatically craft the contents into.
An example would be using the Auto-smelt to turn ore into ingots and then, when enough ingots are available, the auto-craft would turn them into blocks.

**Comment by hgt3345** [comment]
>"An example would be using the Auto-smelt to turn ore into ingots and then, when enough ingots are available, the auto-craft would turn them into blocks."

Kind of an odd example considering you can do this with the compression upgrade, but regardless, I keep running out of sticks and they're a hassle to craft, would like this indeed

**Comment by BlackWolf101** [comment]
My suggestion would be to make an advanced version of the crafting upgrade for autocrafting. My thoughts for using the upgrade would be to use an ore hammer to create dusts automatically so I can use the autosmelt upgrade. All ores get turned into compressed blocks automatically while mining with just a few upgrades (ie. backpack dedicated to mining).
